### PR TITLE
chore: add docker-compose for ee with alert engine

### PR DESCRIPTION
**Pull request by ytvnr**

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-fopcpxbnau.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/docker-compose-ae/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: docker/Makefile [hunk 1]
```diff
@@ -47,6 +48,9 @@ mongodb: ## Run API Management with MongoDB as database.
 keycloak: ## Run API Management with MongoDB as database.
 	cd ./quick-setup/keycloak && docker-compose down -v && docker-compose pull && docker-compose up -d
 
+ee-with-alert-engine: ## Run API Management Enterprise Edition with Alert Engine. ⚠️ No nightly image for this for the moment. do not forget to suffix version with '-ee'. You can
```
Comment (marcambier) [inline docker/Makefile]:
```suggestion
ee-with-alert-engine: ## Run API Management Enterprise Edition with Alert Engine. ⚠️ No nightly image for this for the moment. Do not forget to suffix version with '-ee'.
```

**Review by marcambier — APPROVED** [review]
Just a minor suggestion about a comment typo

**Pull request merged**